otus-02/src/otus_02/homework/pangram.clj

```diff
@@ -1,6 +1,13 @@
 (ns otus-02.homework.pangram
-  (:require [clojure.string :as string]))
+  (:require [clojure.string :as string]
+            [clojure.set :refer [difference]]))
 
+(def ^:private alpha (set "abcdefghijklmnopqrstuvwxyz"))
 
-(defn is-pangram [test-string])
-
+(defn is-pangram
+  [test-string]
+  (->> test-string
+       (string/lower-case)
+       (set)
+       (difference alpha)
```

Comment:
count (= 26) is much easier and do not need inport set & difference

Reply:
Wow. Elegant. Thanks.